bin/next

```diff
@@ -5,6 +5,24 @@ import { spawn } from 'cross-spawn'
 import { watchFile } from 'fs'
 import pkg from '../../package.json'
 
+if (pkg.peerDependencies) {
+  let peerDependencyUnavailable = false
+
+  Object.keys(pkg.peerDependencies).forEach(dependency => {
+    try {
+      // When 'npm link' is used it checks the clone location. Not the project.
+      require.resolve(dependency)
+    } catch (err) {
+      console.log(`${dependency} not found. Please install ${dependency} using 'npm install ${dependency}'`)
+      peerDependencyUnavailable = true
+    }
+  })
+
+  if (peerDependencyUnavailable) {
```

Comment:
@sakulstra has a good point.
We still don't use preact in the server that's where we need it most.

Anyway, @timneutkens we shouldn't kill the process. Just warn them is what we needed.